### PR TITLE
Testerina and some open api tool help text improvements 

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
This adds the suggested changes in https://github.com/ballerina-platform/ballerina-lang/pull/18829

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 1]
```diff
@@ -1,30 +1,32 @@
 NAME
-       ballerina openapi gen-client - generate a mock Ballerina client for a given OpenAPI contract
+       ballerina openapi gen-client - generate mock Ballerina client for a given OpenAPI contract
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]: we need the "a" right?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
-       support limited set of functionality.
+       Note: This is an experimental tool which only supports a
+       limited set of functionality.
 
 
 OPTIONS
        module
-           [ Optional ] This value will be used as the module name when the client is
-           generated.
+           [ Optional ] This value will be used as the module name when the
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion
           [Optional] The module name to use when generating the client.
```

We can remove L18 then.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
+           client is generated.
 
        client-name
-           This value will be used as the client name when the client is generated.
+           This value will be used as the client name when the client is
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion
           The client name to use when generating the client.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 4]
```diff
@@ -1,16 +1,16 @@
 NAME
-       ballerina openapi  gen-contrac - generate an OpenApi contract for a given Ballerina service
+       ballerina openapi  gen-contract - generate OpenApi contract for a given Ballerina service
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: again we need the "an" right?

```suggestion
       ballerina openapi  gen-contract - generate an OpenAPI contract for a given Ballerina service
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 5]
```diff
@@ -29,14 +29,15 @@ OPTIONS
            and the service name.
 
        -o  <contract-file>
-           This option will specify the output location of the generated contract.
+           This option specify the output location of the generated contract.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           The output location of the generated contract.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 6]
```diff
@@ -29,14 +29,15 @@ OPTIONS
...
 
        -s
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: Is this still used? Shall we verify? @DimalChandrasiri

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 7]
```diff
@@ -29,14 +29,15 @@ OPTIONS
...
+           If this option is provided, the generated contract will
+           not be bound to the given ballerina service.
 
 
 EXAMPLES
-       Get help for Ballerina  gen-contrac command
+       Get help for Ballerina  gen-contract command
        $ ballerina openapi gen-contract -h
 
        Generate an OpenApi contract for a service named hello
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
       Generate an OpenAPI contract for a service named hello
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help (hunk 6 above)
Comment (rdhananjaya) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: +1, how ever OpenApiGenContractCmd does use it

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 8]
```diff
@@ -1,43 +1,43 @@
...
 
-OPTIONS
        module
-           [ Optional if ballerina file is provided ] If provided, the contract
-           will be generated using given module name.
+           [Optional] The module name to use when generating the client.
 
        service-name
-           Service name to generate the openapi contract.
+           The client name to use when generating the client.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: Is this correct? Shouldn't it be service name?

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 9]
```diff
@@ -1,43 +1,43 @@
...
            and the service name.
 
        -o  <contract-file>
-           This option will specify the output location of the generated contract.
+           The output location of the generated contract.
 
        -s
-           If this option is provided, the generated contract will not be bound to the given ballerina service.
+           If this option is provided, the generated contract will
+           not be bound to the given ballerina service.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]:
```suggestion
           not be bound to the given Ballerina service.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help [hunk 10]
```diff
@@ -1,43 +1,43 @@
...
 
 
 EXAMPLES
-       Get help for Ballerina  gen-contrac command
+       Get help for Ballerina  gen-contract command
        $ ballerina openapi gen-contract -h
 
-       Generate an OpenApi contract for a service named hello
+       Generate an OpenAPI contract for a service named hello
        $ ballerina openapi gen-contact hello -i hello_service.bal
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: missing a new line.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help (hunk 8 above)
Comment (rdhananjaya) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help]: Oops! yup will fix it

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help [hunk 11]
```diff
@@ -1,35 +1,38 @@
...
+       Note: This is an experimental tool which only supports a
+       limited set of functionality.
 
 
 OPTIONS
        module
-           [ Optional ] This value will be used as the module name when the client is
-           generated.
+           [Optional] The module name to use when generating the client.
+           client is generated.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help]:
```suggestion
```

**Pull request merged**